tool/site/client/package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "EPSAT-Client",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
can we double check these changes? maybe rerun npm install on version 6 of npm

Reply:
Think I've rolled this back using npm v6